apiutils/task.go

```diff
@@ -0,0 +1,21 @@
+package apiutils
```

Comment:
(it's literally 4 lines and it's best practice to just copy around small functions)